openapi_python_client/__init__.py

```diff
@@ -8,10 +8,12 @@
 from pathlib import Path
 from typing import Any, Dict, Optional
 
+import stringcase
```

Comment:
Looks like you didn't end up using this, I assume because you used your fancy util version instead.
```suggestion

```